pocs/utils/database.py

```diff
@@ -46,6 +57,7 @@ def __init__(self, db='panoptes', host='localhost', port=27017, connect=False):
         # Get the mongo client
         self._client = get_shared_mongo_client(host, port, connect)
 
+        # Pre-defined list of collections that are valid.
```

Comment:
Is self.collections intended for use by the caller?

Reply:
I didn't think it needed to be private in case, but I suppose it could be.